rbi/sord.rbi

```diff
@@ -65,7 +65,7 @@ module Sord
         kind: Symbol,
         msg: String,
         item: YARD::CodeObjects::Base,
-        indent_level: Integer
+        indent_level: Integer,
```

Comment:
Parlour (or Sord?) outputs params with commas on every line when multi-line. It's technically valid, but I don't like it.

Reply:
I have mixed feelings about this too, to be honest - I might alter this on the Parlour side.

Reply:
Opened a PR to fix this :)